contracts/Core/StakeManager.sol

```diff
@@ -103,7 +103,6 @@ contract StakeManager is Initializable, ACL, StakeStorage, StateManager, Pause {
         uint256 amount
     ) external initialized checkEpochAndState(State.Commit, epoch, parameters.epochLength()) whenNotPaused {
         require(stakers[stakerId].acceptDelegation, "Delegetion not accpected");
-        require(stakers[stakerId].tokenAddress != address(0x0000000000000000000000000000000000000000), "Staker has not staked yet");
```

Comment:
what if staker stakes and later withdraws his stake? would acceptDelegation  still be true?

Reply:
> what if staker stakes and later withdraws his stake? would acceptDelegation still be true?

yes if staker stakes and setDelegationAcceptance to true and then withdraws then still for that address delegationAcceptance will be true.

Reply:
Addressed in #366 366